default/stringtables/en.txt

```diff
@@ -4628,14 +4628,15 @@ Species Traits
-Also, all starlane entries in the occupied system become guarded by the blockading force, so if hostile forces enter the system, they can exit only by the starlane entry by which they arrived. If more fleets from the same hostile empire arrive from different starlane entries, these entries will become available as an exit as well. If all hostile fleets are destroyed before new fleets arrive, the previously opened exits will become closed again.
-If fleets from two or more hostile empires arrive at an unoccupied system at the same time, neither such empire will establish system control over each other, although they will both establish control against all other empires.
-Space monsters can also establish system control. However, if an empire fleet arrives and stays at the system for one full turn, the monster's control of the system will be broken. If a space monster and an empire fleet arrive at the same time, the empire fleet will be able to establish system control despite the monsters presence.'''
+SYSTEM_BLOCKADE_TITLE
+System Blockade
+
+SYSTEM_BLOCKADE_TEXT
+'''When an armed fleet set to aggressive mode arrives at a system currently unoccupied by hostile fleets, it establishes a blockade. If this happens, [[metertype METER_SUPPLY]] propagation is interrupted for all other empires, and industry generated on hostile planets within the system can only be used on the respective planet. If hostile planets come under attack, their meter growth (e. g. ground troops) will be suspended.
+Also, all starlane entries in the blockaded system become guarded by the blockading force, so if hostile forces enter the system, they can exit only by the starlane entry by which they arrived. However, if more fleets from the blockaded empire arrive by different starlane entries than the previous one, these entries will become available as an exit as well. If at one point all armed aggressive fleets owned by the blockaded empire are destroyed before new fleets arrive, the previously opened exits will become closed again.
+Allied fleets do not have a direct impact on supply blocks and starlane availability apart from helping with eliminating the blockading empire's forces.
+If fleets from two or more hostile empires arrive at an unblockaded system at the same time, neither such empire will establish a system blockade over each other, although they will establish a blockade against all other hostile empires that arrive at a later time.
+Space monsters can also establish blockades. However, if an empire fleet becomes blockaded by a space monster, it will be able to evade the blockade after staying at the system for one full turn and leave the system by any starlane exit. Also, if a space monster and an empire fleet arrive at the same time, the empire fleet will be able to act first and establish a blockade against the monster.'''
```

Comment:
Empire fleets can leave after one turn in a system with monsters, but the supply propagation restrictions remain as long as the monster is present?